Dockerfile.rocm

```diff
@@ -25,8 +25,8 @@ ARG ARG_PYTORCH_ROCM_ARCH="gfx90a;gfx942"
 ENV PYTORCH_ROCM_ARCH=${ARG_PYTORCH_ROCM_ARCH}
 
 # Install some basic utilities
-RUN apt-get update && apt-get install python3 python3-pip -
-RUN apt-get update && apt-get install -y \
+RUN apt-get update -q -y --force-yes && apt-get install -q -y --force-yes python3 python3-pip -
+RUN apt-get update -q -y --force-yes && apt-get install -q -y --force-yes \
```

Comment:
Do we want the force-yes? Could be better to catch the future errors that it would mask

Reply:
well, imo, it is nice to use latest and greatest in containers. Otherwise no reason to update. Right?

Reply:
force-yes means suppress incompatibility and other warning messages. To answer yes to the default apt-get prompt you just need the -y